elements/x-tooltip/x-tooltip.js

```diff
@@ -92,8 +92,16 @@ class XTooltipElement extends HTMLElement {
         clientRect || this._element.getBoundingClientRect();
       this._x = elementClientRect.left;
 
+      let pos = Settings.get(Settings.keys.BOOKMARKS_DRAWER_POSITION);
+
       const tooltipClientRect = this.$tooltip.getBoundingClientRect();
-      const x = this._x - (tooltipClientRect.width - 10);
+
+      let x = this._x - (tooltipClientRect.width - 10);
+
+      if (pos === Settings.enums.BookmarkDrawerPositions.LEFT) {
+        x = this._x + 54;
```

Comment:
This `54` seems like a bit of a magic number.  Could you instead of write it in terms of `elementClientRect.width` and the `TOOLTIP_X_OFFSET`?